src/interfaces/docker_interface.py

```diff
@@ -295,22 +255,19 @@ def add_services(self, tank, services):
                     "BUILD_ARGS": f"{tank.DEFAULT_BUILD_ARGS + tank.extra_build_args}",
                 },
             }
+            services[tank.container_name]['build'] = build
+            self.copy_configs(tank)
         else:
-            # assume it's a release version, get the binary
-            build = {
-                "context": str(tank.config_dir),
-                "dockerfile": str(tank.config_dir / DOCKERFILE_NAME),
-                "args": {
-                    "ARCH": get_architecture(),
-                    "BITCOIN_URL": "https://bitcoincore.org/bin",
-                    "BITCOIN_VERSION": f"{tank.version}",
-                },
-            }
-        # Add the bitcoind service
+            image = f"{DOCKER_REGISTRY}:{tank.version}"
+            services[tank.container_name]['image'] = image
+        # Add common bitcoind service details
         services[tank.container_name].update(
             {
                 "container_name": tank.container_name,
-                "build": build,
+                "environment": {
+                    "WARNET": "1",
```

Comment:
what is the `WARNET` arg for?

Reply:
This is used in entrypoint.sh to optionally copy over our custom warnet torrc

https://github.com/willcl-ark/warnet/blob/f6cb1fc11e6e7b7fcf1c1e66c259abd3e6ca1b6b/src/templates/entrypoint.sh#L12-L14

This means that in the end, passing env var `TOR=1` to the tank will enable tor (regular) and `TOR=1 WARNET=1` will use warnet tor (I'm not done with it yet!)

Reply:
It doesn't matter, but it's incorrectly set here, as we're not using TOR=1